DOCSP-30318: Node.js driver v5.6.0 release (#700)
mongodb/docs-node: source/whats-new.txt

* :ref:`Version 3.7 <version-3.7>`
* :ref:`Version 3.6 <version-3.6>`

.. _version-5.6:

What's New in 5.6
-----------------

The {+driver-short+} v5.6 release includes the following features:

- The driver now supports Node.js v20.

> - The driver can return a cursor as the response to a server command by
>   calling the ``runCursorCommand()`` method. To learn more about this feature,

Reviewer: ```suggestion
- The driver can return a cursor as the response to a server command when you call the
   the ``runCursorCommand()`` method. To learn more about this feature,
```

>   see the `runCursorCommand API documentation <{+api+}/classes/Db.html#command>`__.

Reviewer: S: link to [this API documentation](https://mongodb.github.io/node-mongodb-native/5.6/classes/Db.html#runCursorCommand) instead of the command type documentation

>   see the `runCursorCommand API documentation <{+api+}/5.6/classes/Db.html#command>`__.

Reviewer: I: the link 404s

S: I think the /5.6/ isn't needed

Author: Strange, my last commit didn't seem to go through.

- Support for specifying time series collection creation options
  `bucketMaxSpanSeconds <{+api+}/interfaces/TimeSeriesCollectionOptions.html#bucketMaxSpanSeconds>`__
  and

>   `bucketRoundingSeconds <{+api}/interfaces/TimeSeriesCollectionOptions.html#bucketRoundingSeconds>`__.

Reviewer: I: this link is 404ing

S: add missing + in the source constant reference

Author: Weird, thought I checked. Good catch!

> - Support for specifying time series collection creation options
>   `bucketMaxSpanSeconds <{+api+}/interfaces/TimeSeriesCollectionOptions.html#bucketMaxSpanSeconds>`__
>   and
>   `bucketRoundingSeconds <{+api}/interfaces/TimeSeriesCollectionOptions.html#bucketRoundingSeconds>`__.

Reviewer: ```suggestion
- Support for the following options for creating time series collections:
   -`bucketMaxSpanSeconds <{+api+}/interfaces/TimeSeriesCollectionOptions.html#bucketMaxSpanSeconds>`__
   - `bucketRoundingSeconds <{+api}/interfaces/TimeSeriesCollectionOptions.html#bucketRoundingSeconds>`__
```

Author: I was debating between the two, but didn't see much advantage in listing them separately since there's no accompanying text. Do you think it's necessary to present as a list?

Reviewer: It's not necessary, but as it is the phrase "specifying time series collection creation options" is long and confusing. I thought it could be helpful to break the phrasing up a bit to make it more clear

Author: Discussed on Slack

  To learn more about these time series collection options, see the
  `Set Granularity for Time Series Data <https://www.mongodb.com/docs/upcoming/core/timeseries/timeseries-granularity/#set-granularity-for-time-series-data>`__

>   Server Manual documentation.

Reviewer: S: decapitalize manual in this line (I think this is what we agreed to going forwards?)

Author: Thanks for the reminder!
